### PR TITLE
#48 url generated on elm PB when user clicks publish

### PR DESCRIPTION
**Pull request by naazy**
#48 url generated on elm PB when user clicks publish

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 (function(){
   var node = document.querySelector("div");
   var app = Elm.Main.embed(node, {
-      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : ""
+      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : "",
```
Comment (roryc89) [inline main.js]: does the npm module still work if `/events/` isn't in the url? Could be a case of overly tight coupling.

File: src/MainModel.elm [hunk 2]
```diff
@@ -68,7 +72,9 @@ initialModel =
     , datePickerClosed = True
     , pickedDates = initialDates
     , pickedTracks = []
+    , pickedColumns = []
     , datesWithSessions = [ { date = DateWithoutTime 2017 1 1, sessions = initialSessions } ]
+    , host = "localhost:8000"
```
Comment (roryc89) [inline src/MainModel.elm]: Is there a point in setting here this as it is immediately overwritten by  init?

File: main.js (hunk 1 above)
Comment (roryc89) [inline main.js]: Though I can see this wasn't added in this PR so we can address it in the next PR to be made to elm-program-builder

File: src/TableView.elm [hunk 3]
```diff
@@ -23,7 +23,7 @@ view model =
                     [ thead []
                         [ tr []
                             (defaultHeaders
-                                ++ (List.map viewColumnHeader model.columns)
+                                ++ (model.columns |> List.sortBy .id |> List.map viewColumnHeader)
```
Comment (roryc89) [inline src/TableView.elm]: Really like the use of pipeline

File: main.js (hunk 1 above)
Comment (naazy) [inline main.js]: Ok. Let's take this conversation here: https://github.com/OxfordAbstracts/elm-program-builder/issues/51

File: src/MainModel.elm (hunk 2 above)
Comment (naazy) [inline src/MainModel.elm]: good point

**Pull request merged**